### PR TITLE
Add is_test db config to test harness

### PR DESCRIPTION
**Pull request by terencecho**
Sets the is_test flag to true in the config to bypass the SSL modification for cloud acceptance tests.

Fixes issues first caused by https://github.com/airbytehq/airbyte/pull/15932, where the original change caused acceptance test failures for legacy versions of postgres.

### CODE REVIEW

**Comment by terencecho** [comment]
@davinchia this previous pr: https://github.com/airbytehq/airbyte/pull/15932 that you approved caused some legacy acceptance tests to fail, so i reverted and made changes here. basically if it's a test using an older postgres image, don't include the `IS_TEST` in the config.

**Comment by davinchia** [comment]
@terencecho oh interesting - I'm guessing it's because the older images don't have this option available.

File: airbyte-test-utils/src/main/java/io/airbyte/test/utils/AirbyteAcceptanceTestHarness.java [hunk 1]
```diff
@@ -628,11 +625,12 @@ private Map<Object, Object> localConfig(final PostgreSQLContainer psql,
     dbConfig.put(JdbcUtils.PORT_KEY, psql.getFirstMappedPort());
     dbConfig.put(JdbcUtils.DATABASE_KEY, psql.getDatabaseName());
     dbConfig.put(JdbcUtils.USERNAME_KEY, psql.getUsername());
-    // Some database docker images labeled strict-enforce do not contain an option to ssl off, so it is
-    // not included in the schema.
-    if (!strictEnforce) {
-      dbConfig.put(JdbcUtils.SSL_KEY, false);
+
+    // bypasses the SSL modification for cloud acceptance tests
```
Comment (davinchia) [inline airbyte-test-utils/src/main/java/io/airbyte/test/utils/AirbyteAcceptanceTestHarness.java:629]: can we add here explaining why this bypass is useful?

File: airbyte-test-utils/src/main/java/io/airbyte/test/utils/AirbyteAcceptanceTestHarness.java [hunk 2]
```diff
@@ -145,6 +145,7 @@ public class AirbyteAcceptanceTestHarness {
   public static final String AWESOME_PEOPLE_TABLE_NAME = "awesome_people";
 
   private static final String DEFAULT_POSTGRES_INIT_SQL_FILE = "postgres_init.sql";
+  private static final String IS_TEST = "is_test";
```
Comment (davinchia) [inline airbyte-test-utils/src/main/java/io/airbyte/test/utils/AirbyteAcceptanceTestHarness.java:148]: nit: should add the comment we had back :)

**Review by davinchia — APPROVED** [review]
Appreciate you following up here!

Reminder to add comments for readability. Feel free to merge after you do. Thanks.

**Pull request merged**